generators/wp/templates/wp-config.php

```diff
@@ -101,9 +102,17 @@
 
 /* That's all, stop editing! Happy blogging. */
 
+/** Hide PHP errors */
+if ( !WP_DEBUG ) {
+	ini_set( 'display_errors', 0 );
+}
+
 /** Absolute path to the WordPress directory. */
 if ( !defined('ABSPATH') )
-	define('ABSPATH', dirname(__FILE__) . '/');
+	define( 'ABSPATH', dirname(__FILE__) . '/' );
 
 /** Sets up WordPress vars and included files. */
-require_once(ABSPATH . 'wp-settings.php');
+require_once( ABSPATH . 'wp-settings.php' );
+
+/** Disable the Plugin and Theme Editor */
+define( 'DISALLOW_FILE_EDIT', true );
```

Comment:
Why don't we add this in else clause so this can be easily changed in wp-config-local.php?